README.md

```diff
@@ -330,6 +332,24 @@ public class MyJobExecutorCustomizer implements JobExecutorCustomizer {
 }
 ```
 
+## Add Process Engine Plugins
+Every class that implements `ProcessEnginePlugin` gets automatically added to the engine on start. 
```

Comment:
* "class" -> "bean"
* "implements" -> "implements the interface"
* "engine" -> "process engine"